gitlab/const.py

```diff
@@ -33,3 +33,40 @@
 NOTIFICATION_LEVEL_GLOBAL = "global"
 NOTIFICATION_LEVEL_MENTION = "mention"
 NOTIFICATION_LEVEL_CUSTOM = "custom"
+
+_SEARCH_SCOPE_PROJECTS = "projects"
+_SEARCH_SCOPE_ISSUES = "issues"
+_SEARCH_SCOPE_MERGE_REQUESTS = "merge_requests"
+_SEARCH_SCOPE_MILESTONES = "milestones"
+_SEARCH_SCOPE_WIKI_BLOBS = "wiki_blobs"
+_SEARCH_SCOPE_COMMITS = "commits"
+_SEARCH_SCOPE_BLOBS = "blobs"
+_SEARCH_SCOPE_USERS = "users"
+
+SEARCH_SCOPE_GLOBAL_PROJECTS = _SEARCH_SCOPE_PROJECTS
+SEARCH_SCOPE_GLOBAL_ISSUES = _SEARCH_SCOPE_ISSUES
+SEARCH_SCOPE_GLOBAL_MERGE_REQUESTS = _SEARCH_SCOPE_MERGE_REQUESTS
+SEARCH_SCOPE_GLOBAL_MILESTONES = _SEARCH_SCOPE_MILESTONES
+SEARCH_SCOPE_GLOBAL_SNIPPET_TITLES = "snippet_titles"
+SEARCH_SCOPE_GLOBAL_WIKI_BLOBS = _SEARCH_SCOPE_WIKI_BLOBS
+SEARCH_SCOPE_GLOBAL_COMMITS = _SEARCH_SCOPE_COMMITS
+SEARCH_SCOPE_GLOBAL_BLOBS = _SEARCH_SCOPE_BLOBS
+SEARCH_SCOPE_GLOBAL_USERS = _SEARCH_SCOPE_USERS
+
+SEARCH_SCOPE_GROUP_PROJECTS = _SEARCH_SCOPE_PROJECTS
```

Comment:
Why is the duplication needed?

Reply:
Personally, +1 vote for this approach, I like your last suggestion in this comment, it's much more concise than in the original commit :)